src/etc/di.xml

```diff
@@ -34,13 +34,13 @@
             <argument name="validatorFile" xsi:type="object">ScandiPWA\QuoteGraphQl\Model\Product\Option\Type\File\ValidatorFile\Proxy</argument>
         </arguments>
     </type>
-    <virtualType name="ScandiPWA\Router\Controller\ConfigurableRouter" type="ScandiPWA\Router\Controller\Router">
-        <arguments>
-            <argument name="ignoredURLs" xsi:type="array">
-                <item name="sales" xsi:type="string">^/sales/.*</item>
-            </argument>
-        </arguments>
-    </virtualType>
+<!--    <virtualType name="ScandiPWA\Router\Controller\ConfigurableRouter" type="ScandiPWA\Router\Controller\Router">-->
+<!--        <arguments>-->
+<!--            <argument name="ignoredURLs" xsi:type="array">-->
+<!--                <item name="sales" xsi:type="string">^/sales/.*</item>-->
+<!--            </argument>-->
+<!--        </arguments>-->
+<!--    </virtualType>-->
```

Comment:
Is it worth commenting these lines? Why not remove them altogether?

Reply:
Those lines where added for supporting file uploading on products, but dev who was making it is not even working anymore so no idea why sales urls where  ignored. With PM suggested to save it and in case when file upload will be broken cuz specific url is not ignored, return it.

But actually as practice shows commented code stays commented forever... So I assume you right, better to remove it at all.